sumdbaudit/client/sumdb_test.go

```diff
@@ -67,7 +67,7 @@ func TestLeavesAtOffset(t *testing.T) {
 			t.Errorf("expected string terminating in newline, got: %x", l)
 		}
 		expStart := "golang.org/x/"
-		if got, want := fmt.Sprintf("%s", l[:len(expStart)]), expStart; got != want {
+		if got, want := string(l[:len(expStart)]), expStart; got != want {
```

Comment:
Does this need the `string()` cast?

Reply:
Yes, because `want` is a string, and `l` is `[]byte`. You can't `!=` `string` and `[]byte`.